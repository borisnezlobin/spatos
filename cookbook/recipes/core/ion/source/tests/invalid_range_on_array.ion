let array = [ 1 2 3 4 5 ]
echo @array[5..12]
