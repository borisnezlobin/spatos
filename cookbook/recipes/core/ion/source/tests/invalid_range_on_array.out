ion: expansion error: Invalid range between (5,12) for sequence with length 5
